scripts/add-yearn-vaults.js

```diff
@@ -0,0 +1,56 @@
+const hre = require("hardhat");
+const ethers = hre.ethers;
+const clonedeep = require("lodash.clonedeep");
+
+const ConfigLoader = require("./utils/configurator-loader.js");
+const MultisigExecutor = require("./utils/multisigexecutor.js");
+
+const TRUSTLIST = 0;
+
+async function main() {
+  const configLoader = new ConfigLoader(hre.network.name);
+  const config = await configLoader.load();
+  const configUpdate = clonedeep(config);
+
+  const [deployer] = await ethers.getSigners();
+  console.log(`Deployer is ${deployer.address} with balance ETH ${ethers.utils.formatEther(await ethers.provider.getBalance(deployer.address))}`);
+
+  const dappRegistry = await ethers.getContractAt("DappRegistry", config.dappRegistry.address);
+  const yearnV2Filter = await ethers.getContractAt("YearnV2Filter", config.yearn.v2.filter);
+  const registryOwner = await dappRegistry.registryOwners(TRUSTLIST);
```

Comment:
Registry owner is not needed if we don't check that it is the multisig, so maybe we should add a condition and exit if it is not the multisig.